jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/ExplorerToolWindow.kt

```diff
@@ -65,70 +71,67 @@ class ExplorerToolWindow(private val project: Project) : SimpleToolWindowPanel(t
 
         project.messageBus.connect().subscribe(ProjectAccountSettingsManager.ACCOUNT_SETTINGS_CHANGED, this)
 
-        updateModel()
+        treePanelWrapper.setContent(errorPanel)
```

Comment:
Is `errorPanel` the "select creds please" panel or something else?

Reply:
You are correct